### PR TITLE
Initialize variables at top of scope for VS2010

### PR DESCRIPTION
**Pull request by pshipton**
The Windows compiles fail without this change.
https://ci.eclipse.org/openj9/job/Pipeline-OpenJDK8-Acceptance/51

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins test sanity,sanity.openjdk win,win32 jdk8 depends ibmruntimes/openj9-openjdk-jdk8#openj9-staging

**Comment by pshipton** [comment]
jenkins test sanity,sanity.openjdk win,win32 jdk8

File: jdk/src/share/back/classTrack.c [hunk 1]
```diff
@@ -81,13 +87,13 @@ cbTrackingObjectFree(jvmtiEnv* jvmti_env, jlong tag)
 struct bag *
 classTrack_processUnloads(JNIEnv *env)
 {
+    struct bag* deleted = deletedSignatures;
```
Comment (keithc-ca) [inline jdk/src/share/back/classTrack.c:90]: Reading `deletedSignatures` must be within the locked region.

Comment (pshipton) [reply jdk/src/share/back/classTrack.c]: Fixed.

**Comment by pshipton** [comment]
Note the previous PR testing passed.
https://ci.eclipse.org/openj9/view/Pull%20Requests/job/PullRequest-OpenJDK8/103/

**Pull request merged**